### ISSUE TITLE
Does not start on F33/F34

### ISSUE THREAD

**Issue by bocekm** [flathub/com.synology.SynologyDrive#13]
Reproducer:

- install it on Fedora 33 through `flatpak install flathub com.synology.SynologyDrive`
- run it through the Fedora "desktop" icon, create a simple backup job and close it
- try to run it again through the "desktop" icon => nothing happens
- try to run it through `flatpak run com.synology.SynologyDrive` and see the error below
- try `flatpak repair com.synology.SynologyDrive`, uninstall & install => doesn't help

Error:
```
$ flatpak run com.synology.SynologyDrive
lib path = '/app/extra/SynologyDrive/lib/plugins'
Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
Qt: Session management error: None of the authentication protocols specified are supported
Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
Gtk-Message: 17:59:55.161: Failed to load module "canberra-gtk-module"
Gtk-Message: 17:59:55.161: Failed to load module "pk-gtk-module"
Gtk-Message: 17:59:55.161: Failed to load module "canberra-gtk-module"
Gtk-Message: 17:59:55.161: Failed to load module "pk-gtk-module"
Qt: Session management error: None of the authentication protocols specified are supported
QLayout: Attempting to add QLayout "" to TutorialPanel "tutorial_panel", which already has a layout
QLayout: Attempting to add QLayout "" to TutorialPanel "tutorial_panel", which already has a layout
QLayout: Attempting to add QLayout "" to TutorialPanel "tutorial_panel", which already has a layout
QLayout: Attempting to add QLayout "" to TutorialPanel "tutorial_panel", which already has a layout
QLayout: Attempting to add QLayout "" to TutorialDiagram "", which already has a layout
QLayout: Attempting to add QLayout "" to TutorialDiagram "", which already has a layout
sh: line 1: /sbin/udevadm: No such file or directory
sh: line 1: /sbin/ifconfig: No such file or directory
```

**Comment by bryanpaget** [comment]
Does not start on Fedora 36 Silverblue. 

```
[bryan@fedora ~]$ flatpak run com.synology.SynologyDrive
[bryan@fedora ~]$ Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
Qt: Session management error: None of the authentication protocols specified are supported
Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
Gtk-Message: 08:52:04.361: Failed to load module "canberra-gtk-module"
Gtk-Message: 08:52:04.361: Failed to load module "canberra-gtk-module"
Qt: Session management error: None of the authentication protocols specified are supported
[bryan@fedora ~]$ QT_QPA_PLATFORM=wayland flatpak run com.synology.SynologyDrive
[bryan@fedora ~]$ Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
qt.qpa.plugin: Could not find the Qt platform plugin "wayland" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: xcb.

```

**Comment by lil5** [comment]
I've tested in gnome boxes fedora silverblue gnome 37 and it work please update and if it doesn't work message me back here and I'll reopen 

Fixed as of 3.2.1
